main/global-accelerators.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+const {globalShortcut} = require('electron');
+const ipc = require('electron-better-ipc');
+const shortcutToAccelerator = require('../renderer/utils/shortcut-to-accelerator');
+const store = require('./common/settings');
+const {openCropperWindow} = require('./cropper');
+
+const registerShortcut = shortcut => {
```

Comment:
Could have this accept two arguments, `shortcut` and `action` so we can re-use it if we have different shortcuts in the future. And then we can do `registerShortcut(cropperShortcut, openCropperWindow)` in the `registerFromStore` method

Reply:
yep, just didnt want to add too much features here since there is only one shortcut now